CodeGeneration/Sources/SyntaxSupport/GrammarGenerator.swift

```diff
@@ -60,4 +65,25 @@ struct GrammarGenerator {
       .map { " - `\($0.varOrCaseName)`: \(generator.grammar(for: $0))" }
       .joined(separator: "\n")
   }
+
+  /// Generates a markdown string describing possible choices for the given child
+  /// token.
+  static func childTokenChoices(for choices: [TokenChoice]) -> String {
+    let grammar = GrammarGenerator()
+    
+    if choices.count == 1 {
+      return """
+        ### Tokens
+
+        For syntax trees generated by the parser, this is guaranteed to be \(grammar.grammar(for: choices.first!)) 
```

Comment:
And a very minor nitpick.

```suggestion
        For syntax trees generated by the parser, this is guaranteed to be \(grammar.grammar(for: choices.first!)).
```